### ISSUE TITLE
GotNotify: Version mismatch! when upgrading 

### ISSUE THREAD

**Issue by JayWStapleton** [tailscale/tailscale#4019]
### What is the issue?

```
Tailscale Error
Tailscale backend error:
GotNotify: Version mismatch! 
```
After upgrading in Windows

User submitted via forum. 

### Steps to reproduce

If run unattended is selected and the upgrade is done while another user is logged on to the machine. 

### OS

Windows

**Comment by bradfitz** [comment]
@JayWStapleton, MSI or NSIS installer?

The installer should be restarting the GUI.

**Comment by JayWStapleton** [comment]
Customer reports that it's MSI

**Comment by DentonGentry** [comment]
They said: "I click the upgrade notification in the app which auto downloads the new version, so yes, the MSI installer."
https://forum.tailscale.com/t/windows-10-1-14-4-1-16-0-upgrade-backend-error/1199/3

I suspect that is the NSIS installer, despite what their conclusion. It could only be MSI if they had been running 1.21.x.

**Comment by jakedgy** [comment]
I've seen a similar error message when the MacOS client upgrades as well, across multiple versions going back to like 1.16.  Updates are managed through Kinetix/JAMF app store.

**Comment by DentonGentry** [comment]
I'm going to close this as a duplicate of https://github.com/tailscale/tailscale/issues/4091, and expect the fix for that issue will resolve this behavior. The fix will be delivered in 1.24.x.

If similar behavior manifests with releases > 1.24.x please open a new bug, we need to look afresh for a different root cause.
